spark/dl/pom.xml

```diff
@@ -147,6 +147,11 @@
             <artifactId>log4j</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+        <dependency>
```

Comment:
what's this for ?

Reply:
coco annotation raw metadata is in json file. We need a json parser